<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="pushButton_play">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: italic 14pt &quot;URW Palladio L&quot;;
border:2px groove gray;
border-radius:10px 10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>play</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>361</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">margin:o auto;
color: rgb(255, 255, 255);
font: italic 18pt &quot;URW Bookman L&quot;;</string>
   </property>
   <property name="text">
    <string>NOTHING TO SHOW</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>340</y>
     <width>92</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: italic 14pt &quot;URW Palladio L&quot;;
border:2px groove gray;
border-radius:10px 10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>340</y>
     <width>92</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: italic 14pt &quot;URW Palladio L&quot;;
border:2px groove gray;
border-radius:10px 10px;
padding:2px 4px;</string>
   </property>
   <property name="text">
    <string>previous</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>pushButton_play</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
